From e854f357f4037e2c8c781ebd04ec5164a882b9b3 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Thu, 12 Apr 2018 00:45:50 +0200
Subject: [PATCH] Make Qt5Network conditional on BUILD_TESTS

Summary: I did not find it in use somewhere else.

Reviewers: iromanov, sitter, rjvbb

Reviewed By: rjvbb

Subscribers: fvogt

Differential Revision: https://phabricator.kde.org/D12129
---
 CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 28b0169..65b95f8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,7 +59,11 @@ if (Qt5Core_FOUND)
   message(STATUS "Building with Qt5 support")
   # Got from ECM
   # Distros have no ECM. So I just copied required cmake modules.
-  find_package(Qt5Transitional REQUIRED Core Network)
+  if(BUILD_TESTS)
+    find_package(Qt5Transitional REQUIRED Core Network)
+  else()
+    find_package(Qt5Transitional REQUIRED Core)
+  endif()
   include(ECMQt4To5Porting)
 
   include(GNUInstallDirs)
-- 
2.17.1

